content/zh/docs/reference/access-authn-authz/rbac.md

```diff
@@ -2121,7 +2121,7 @@ kubectl create clusterrolebinding permissive-binding \
 {{< warning >}}
 下面的策略允许 **所有** 服务帐户充当集群管理员。
 容器中运行的所有应用程序都会自动收到服务帐户的凭据，
-可以对 API 执行任何操作，包括查看秘钥和修改权限，
+可以对 API 执行任何操作，包括查看 Secrets 和修改权限。
```

Comment:
Since the PR is in general not viewable, can you close this one and reopen a new one?

Reply:
the URL is not working due to too many review comments or sessions. 
https://github.com/kubernetes/website/pull/18947